client/src/components/Login.js

```diff
@@ -1,11 +1,162 @@
+    }
+    if (this.state.password < 1) {
+      isError = true;
+      errors.isErrorPassword = true;
+      errors.passwordError = 'Please enter your password.';
+    }
+    this.setState({
+      ...this.state,
+      ...errors,
+    });
+    return isError;
+  };
+
+  login = () => {
+    const err = this.validate();
+    if (!err) {
+      const { history } = this.props;
+      const inputs = {
+        email: this.state.email,
+        password: this.state.password,
+      };
+      const detailsInputs = this.props.location.data;
+      axios.post('/login', inputs).then(({ data }) => {
+        if (data.success) {
+          if (detailsInputs) {
+            axios.post('/dashboard', detailsInputs).then(addData => {
+              if (addData.data.success) {
+                history.push('/dashboard');
+              } else {
+                history.push('/login');
+              }
+            });
+          } else {
+            history.push('/dashboard');
+          }
+        } else {
+          this.setState({
+            passwordError: 'email or password is incorrect.',
+            isErrorEmail: true,
+            isErrorPassword: true,
+          });
+        }
+      });
+    }
+  };
+
   render() {
     return (
-      <Fragment>
-        <h1>Login</h1>
-      </Fragment>
+      <React.Fragment>
+        <StyleAll>
+          <Stylecontainer>
+            <Stylephoto src={blp} alt="photo" />
+            <Link href={INFO_URL}>
+              <Styleques src={question} alt="info" />
+            </Link>
+          </Stylecontainer>
+          <Title>LOGIN</Title>
+          <StyledForm>
+            <InputContainer>
+              <StyledLabel> Email: </StyledLabel>
+              <StyledInput
+                StyleError={this.state.isErrorEmail}
+                {...this.props}
+                type="text"
+                name="email"
+                errorText={this.state.EmailError}
+                value={this.state.email}
+                onChange={e =>
+                  this.setState({
+                    email: e.target.value,
+                  })
+                }
+              />
+            </InputContainer>
+
+            <InputContainer>
+              <StyledLabel> Password: </StyledLabel>
+              <StyledInput
+                StyleError={this.state.isErrorPassword}
+                {...this.props}
+                type="password"
+                name="password"
+                errorText={this.state.passwordError}
+                value={this.state.password}
+                onChange={e =>
+                  this.setState({
+                    password: e.target.value,
+                  })
+                }
+              />
+            </InputContainer>
+          </StyledForm>
+          <StyledBottom>
+            <LoginButton onClick={this.login}>LOGIN</LoginButton>
+          </StyledBottom>
+          <StyledP>
+            <StyledText>Forget Your Password? </StyledText>
```

Comment:
Be careful, here you have nested two ```h1``` tags, which create this error on the front end console. There should only be one and we should change it to something like a ```h2``` or ```h3```, since ```h1``` tags should only be used as the primary title in the header of page. Thanks!
![Screen Shot 2019-06-17 at 10 59 55 AM](https://user-images.githubusercontent.com/9094166/59596171-18d97a00-90ef-11e9-8667-4a4245e2b8c8.png)